packages/components/core/utils/colors.ts

```diff
@@ -0,0 +1,24 @@
+export const PresetColor = [
+    'red', 
+    'orange', 
+    'gold', 
+    'yellow', 
+    'canary', 
+    'prasinous', 
+    'verdant', 
+    'green', 
+    'cyan', 
+    'sky', 
+    'blue', 
+    'admiral', 
+    'purple', 
+    'magenta',
+    'lime', 
+    'grey'
+] as const
+
+export const isPresetColor = (color?: string): boolean => {
```

Comment:
```suggestion
export const isPresetColor = (color: string | undefined): boolean => {
```

? 和显式的声明为 undefined 是有区别的, 这里显式声明更符合使用场景
- ? 表示可以不传
- undefined 表示必传，但是可以为 undefined